Added put.md in the terms folder of `hashmap` (#1521)
Codecademy/docs: content/kotlin/concepts/hashmap/terms/put/put.md

---
Title: '.put()'

> Description: 'Used to add a key-value pair to a hash map.'

Reviewer: ```suggestion
Description: 'Inserts a key-value pair to a hash map.'
```

---
Title: '.put()'
Description: 'Used to add a key-value pair to a map.'
Subjects:
  - 'Code Foundations'
  - 'Computer Science'
Tags:
  - 'HashMap'
  - 'Data Types'
  - 'Collections'
  - 'Data Structures'
CatalogContent:
  - 'learn-kotlin'
  - 'paths/computer-science'
---

In Kotlin, the `put()` function is used to add a key-value pair to a map. It is a member function of the `MutableMap` interface, which means it can be called on any object that implements this interface, such as a `HashMap` or `LinkedHashMap`.

## Syntax

```pseudo
fun <K, V> MutableMap<K, V>.put(key: K, value: V): V?
```

It takes in two parameters:

> - key: the key of the key-value pair to add or update in the map.
> - value: the value of the key-value pair to add or update in the map.

Reviewer: ```suggestion
- The `key` of the key-value pair to add or update in the map.
- The `value` of the key-value pair to add or update in the map.
```

We can also add on here what the functionality is for the key and value. For example, the key is a unique identifier that is used to access certain values within the hash map and the value is what's actually being stored or the data that associated with the key.

## Example

> To add a key-value pair to a `HashMap` in Kotlin, you can use the `put()` function. For example:

Reviewer 2: ```suggestion
To add a key-value pair to a `HashMap` in Kotlin, the `put()` function can be used. For example:
```

> To add a key-value pair to a `HashMap` in Kotlin, the `put()` function can be used. For example:

Reviewer: ```suggestion
To add a key-value pair to a `HashMap` in Kotlin, the `.put()` method can be used. For example:
```

```kotlin
fun main(){
val map = HashMap<String, Int>()
map.put("apple", 1)
map.put("banana", 2)
map.put("cherry", 3)
}
```

> We can also use the index operator `[]` to add a key-value pair to the `HashMap`. For example:

Reviewer 2: ```suggestion
The index operator `[]` can be used to add a key-value pair to a `HashMap`. For example:
```
